static/js/stream_ui_updates.js

```diff
@@ -162,6 +163,27 @@ export function update_permissions_banner(sub) {
     $settings.find(".stream-settings-tip-container").html(rendered_tip);
 }
 
+export function update_notifications_stream_in_settings() {
```

Comment:
You can add an return statement here if stream creation form is not opened. This way we can avoid doing all this unnecessary updates when the form is not opened.